wallet/src/de/schildbach/wallet/offline/HttpDirectPaymentTask.kt

```diff
@@ -1,117 +0,0 @@
-
-/*
-* Copyright 2013-2015 the original author or authors.
-*
-* This program is free software: you can redistribute it and/or modify
-* it under the terms of the GNU General Public License as published by
-* the Free Software Foundation, either version 3 of the License, or
-* (at your option) any later version.
-*
-* This program is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-* GNU General Public License for more details.
-*
-* You should have received a copy of the GNU General Public License
-* along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
-
-package de.schildbach.wallet.offline
-
-import de.schildbach.wallet.offline.DirectPaymentTask.ResultCallback
-import de.schildbach.wallet_test.R
-import okhttp3.CacheControl
-import okhttp3.MediaType
-import okhttp3.MediaType.Companion.toMediaTypeOrNull
-import okhttp3.Request
-import okhttp3.RequestBody
-import okio.BufferedSink
-import org.bitcoin.protocols.payments.Protos.Payment
-import org.bitcoin.protocols.payments.Protos.PaymentACK
-import org.bitcoinj.protocols.payments.PaymentProtocol
-import org.dash.wallet.common.util.Constants
-import org.slf4j.LoggerFactory
-import java.io.IOException
-
-abstract class HttpDirectPaymentTask(
```

Comment:
These classes are just Kotlin copies of the java classes and as such not very useful. When we rewrite code to Kotlin we should think about how to make use of coroutines and simplify execution flow for the caller.